### PR TITLE
Document TraceConfig

### LINKED ISSUE #1496: Document TraceConfig
Currently SDK configuration, or `TraceConfig` specifically, is not documented. It does have nice javadoc, but SDK users are not always developers, who read javadoc. They may be operations people. Or they may even be users of auto instrumentation java agent. Both of these groups will benefit from a public document describing available configuration.

### PR DESCRIPTION
**Pull request by malafeev**
fix #1496

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -402,6 +403,43 @@ tracerProvider.addSpanProcessor(BatchSpanProcessor.newBuilder(
...
+```
+
+Supported system properties and environment variables:
+
+| System property                  | Environment variable             | Purpose                                            | 
+|----------------------------------|----------------------------------|----------------------------------------------------|       
+| otel.config.sampler.probability  | OTEL_CONFIG_SAMPLER_PROBABILITY  | Sampler which is used when constructing a new span |                        
+| otel.config.max.attrs            | OTEL_CONFIG_MAX_ATTRS            | Max number of attributes per span                  |                        
+| otel.config.max.events           | OTEL_CONFIG_MAX_EVENTS           | Max number of Events per span                      |                        
+| otel.config.max.links            | OTEL_CONFIG_MAX_LINKS            | Max number of Link} entries per span               |
```
Comment (iNikem) [inline QUICKSTART.md:438]: extra `}` after `Link`

File: QUICKSTART.md [hunk 2]
```diff
@@ -402,6 +403,43 @@ tracerProvider.addSpanProcessor(BatchSpanProcessor.newBuilder(
...
+// Update the resulting TraceConfig instance
+OpenTelemetrySdk.getTracerProvider().updateActiveTraceConfig(builder.build());
+```
+
+Supported system properties and environment variables:
+
+| System property                  | Environment variable             | Purpose                                            | 
+|----------------------------------|----------------------------------|----------------------------------------------------|       
+| otel.config.sampler.probability  | OTEL_CONFIG_SAMPLER_PROBABILITY  | Sampler which is used when constructing a new span |                        
+| otel.config.max.attrs            | OTEL_CONFIG_MAX_ATTRS            | Max number of attributes per span                  |
```
Comment (iNikem) [inline QUICKSTART.md:436]: For all MAX attributes it would be useful to say what happens if that limit is exceeded.

**Comment by iNikem** [comment]
Thanks for documentation PRs!

File: QUICKSTART.md (hunk 1 above)
Comment (malafeev) [reply QUICKSTART.md]: fixed

File: QUICKSTART.md (hunk 2 above)
Comment (malafeev) [reply QUICKSTART.md:436]: javadoc doesn't say what happens, I suppose exceeded should be dropped.

Comment (iNikem) [reply QUICKSTART.md:436]: Yes, extra attributes/links/events will be dropped. Too long values will be truncated.

Comment (malafeev) [reply QUICKSTART.md]: updated

File: QUICKSTART.md [hunk 3]
```diff
@@ -402,6 +403,43 @@ tracerProvider.addSpanProcessor(BatchSpanProcessor.newBuilder(
 ).build());
 ```
 
+### TraceConfig
+
+`TraceConfig` associated with `TracerSdkProvider` can be updated via system properties,
```
Comment (jkwatson) [inline QUICKSTART.md:408]:
```suggestion
The `TraceConfig` associated with a `TracerSdkProvider` can be updated via system properties, 
```

**Comment by jkwatson** [comment]
Looks good! Just one tiny update to the text desired. Thanks for the contribution!

**Comment by trask** [comment]
Maybe document the default values? (and if you don't mind, a PR to sync https://github.com/open-telemetry/opentelemetry-java-instrumentation#trace-config)

File: QUICKSTART.md (hunk 3 above)
Comment (malafeev) [reply QUICKSTART.md]: done

**Comment by malafeev** [comment]
@trask I added default values

**Review by thisthat — APPROVED** [review]

**Pull request merged**